test/routes/report/report.test.js

```diff
@@ -751,18 +753,18 @@ describe('/reports/{REPORTID}', () => {
 
   // delete report
   afterAll(async () => {
-    await db.models.report.destroy({where: {id: report.id}, force: true});
```

Comment:
add these back since tests should be passing now

Reply:
I added them back and the test is timing out again

Reply:
are the records actually getting deleted if force: true is removed?

Reply:
sadly they are not. I just did a count before and after the test and it went up by 19. Will confer with @Nithriel on this

Reply:
@Nithriel just confirmed that force: true works on github. Pushing change now